examples/electricity/amount_energy_from_voltage_time_resistance.py

```diff
@@ -0,0 +1,62 @@
+from symplyphysics import (
+    units, convert_to,solve, SI,Quantity, expr_to_quantity
+)
+from symplyphysics import (
+    symbols, Eq, pretty
+)
+from symplyphysics.laws.electricity import amount_energy_from_voltage_time_resistance as joule_lenz_law
+from symplyphysics.definitions import power_is_proportional_voltage_and_current as operate_power
+from symplyphysics.definitions import amount_energy_from_mass_and_temperature as operate_energy
+from symplyphysics.laws.electricity import current_is_proportional_to_voltage as ohm_law
+
+# The household electric kettle heated 0.5 liters of water to boiling.
+# The power of the kettle is 1500 watts. How long will it take for the
+# temperature of the water to drop to a comfortable 35 degrees?
+P_example = units.Quantity('P_example')
+SI.set_quantity_dimension(P_example, units.power)
+SI.set_quantity_scale_factor(P_example, 1500 * units.watt)
+m_example = units.Quantity('m_example')
+SI.set_quantity_dimension(m_example, units.mass)
+SI.set_quantity_scale_factor(m_example, 0.5 * units.kilogram)
+C_example = units.Quantity('C_example')
+SI.set_quantity_dimension(C_example, units.energy / (units.mass * units.temperature))
+SI.set_quantity_scale_factor(C_example, 4200 * units.joule / (units.kilogram * units.kelvin))
+t1_example = units.Quantity('t1_example')
+SI.set_quantity_dimension(t1_example, units.temperature)
+SI.set_quantity_scale_factor(t1_example, 373 * units.kelvin)
```

Comment:
What are these numbers? Check if it possible to convert from Celsius degrees to Kelvin or add an explanation what you are doing here.